src/main/java/com/ss/lms/entity/Book.java

```diff
@@ -1,41 +1,81 @@
 package com.ss.lms.entity;
 
-import java.io.Serializable;
+
+
+import java.lang.annotation.Repeatable;
 import java.util.List;
 
-public class Book implements Serializable{
-	/**
-	 * 
-	 */
-	private static final long serialVersionUID = 5439836585770906208L;
-	private Integer bookId;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+
+@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class,
+property = "bookId")
+@Entity
+@Table(name = "tbl_book")
+public class Book {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "bookId")
+	private Long bookId;
+	
+	@Column(name = "title")
 	private String title;
+	
+	@ManyToOne
+	@JoinColumn(name = "pubId")
 	private Publisher publisher;
+	
+	@ManyToMany(cascade = CascadeType.ALL)
```

Comment:
i was using cascade type.persist because I didnt think we want CascadeType.delete implemented. Since the Book is the "owner" of the relationship wouldn't that make it delete the author every time a book is deleted.

Reply:
Thanks. I will look into it.